evm_arithmetization/src/cpu/kernel/tests/mod.rs

```diff
@@ -1,10 +1,13 @@
 mod account_code;
+#[cfg(feature = "eth_mainnet")]
```

Comment:
This one is the kernel test, not the integration test. I don't know if this is what you're targeting.

Reply:
It requires to be run on `eth_mainnet` because of the beacon roots, hence the deactivation (same for bls12 test relating to the KZG point evaluation precompile)